modules/liquid.nix

```diff
@@ -10,6 +10,7 @@ let
     ${optionalString cfg.testnet "testnet=1"}
     ${optionalString (cfg.dbCache != null) "dbcache=${toString cfg.dbCache}"}
     ${optionalString (cfg.prune != null) "prune=${toString cfg.prune}"}
+    ${optionalString (cfg.validatepegin != true) "validatepegin=0"}
```

Comment:
We can't assume Liquid defaults, right? Either the `cfg.validatepegin` should a `true|false|unset` variable or it should always be added to the config.

Reply:
Yes the line should be something like
```
    ${optionalString (cfg.validatepegin != null) "validatepegin=${if cfg.validatepegin then "1" else "0"}"}
```